updating/updating-disconnected-cluster.adoc

```diff
@@ -1,23 +1,157 @@
+----
++
+----
+$ oc adm release mirror -a ${LOCAL_SECRET_JSON} --to-dir=${REMOVABLE_MEDIA_PATH}/mirror quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE}
+----
++
+[NOTE]
+====
+If the local container registry and the cluster are connected to the mirror host, you can directly push the release images to the local registry and apply the ConfigMap to the   cluster using following command.
+----
+$ oc adm release mirror -a ${LOCAL_SECRET_JSON} --from=quay.io/${PRODUCT_REPO}/${RELEASE_NAME}:${OCP_RELEASE}-${ARCHITECTURE} \
+  --to=${LOCAL_REGISTRY}/${LOCAL_REPOSITORY} --apply-release-image-signature
+----
+If you are using `--apply-release-image-signature` you do not need to create the ConfigMap for image signature verification.
+====
++
+** Take the media to the restricted network environment and upload the images to the local container registry.
++
+----
+$ oc image mirror  -a ${LOCAL_SECRET_JSON} --from-dir=${REMOVABLE_MEDIA_PATH}/mirror 'file://openshift/release:${OCP_RELEASE}*' ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}
+----
+
+=== Create the ConfigMap for image signature verification
+
+The Cluster Version Operator (CVO) in the cluster performs a verification to make sure the signatures of the release images are not tampered. To do that it compares expected VS actual image signatures. For the verification process it needs to know the image signatures. The image signatures are available in link:https://mirror.openshift.com/pub/openshift-v4/signatures/openshift/release[mirror.openshift.com]  and link:https://storage.googleapis.com/openshift-release/official/signatures[Google Cloud Storage (GCS)]. However in a restricted network environment we do not have access to the internet, so we need to create ConfigMap which CVO can use to verify the image signatures.
+
+[NOTE]
+====
+If you are upgrading from an earlier release than 4.4.8, you must use the manual method for creating the ConfigMap.
+
+This is because the `oc` command-line interface (CLI) enhancement is not available in previous releases.
+====
+
+.Procedure
+
+. Use `oc` command-line interface (CLI) to log in to a given cluster which going to be upgraded.
+
++
+. Apply the the mirrored release image signature ConfigMap to the connected cluster
++
+----
+Example:
+
+$ oc apply -f mirror/config/signature-sha256-81154f5c03294534.yaml
+----
+
+=== Perform the upgrade
+
+.Procedure
+
+. Perform the upgrade
++
+----
+Example:
+
+$ oc adm upgrade --allow-explicit-upgrade --to-image ${LOCAL_REGISTRY}/${LOCAL_REPOSITORY}@sha256:81154f5c03294534e1eaf0319bef7a601134f891689ccede5d705ef659aa8c92
+
+The sha256sum value can be found in the image signature ConfigMap in the previous step.
+----
+[NOTE]
+====
+If you are using ImageContentSourcePolicy for the mirror registry you can use the canonical registry name instead of LOCAL_REGISTRY.
+====
+
+include::modules/images-configuration-registry-mirror.adoc[leveloffset=+2]
```

Comment:
Is this included module in the right place?

Reply:
As per http://file.rdu.redhat.com/kalexand/063020/OTA-102/updating/updating-restricted-network-cluster.html I think it is in the right place. Also we can fix it later if we get feedback from users.